forge/ee/lib/sso/index.js

```diff
@@ -323,8 +324,27 @@ module.exports.init = async function (app) {
                         }
                     }
                 }
+                if (providerOpts.groupAdmin && providerOpts.groupAdminName === ga) {
+                    adminGroup = true
+                }
             })
 
+            if (user.admin && !adminGroup) {
+                if (!Object.hasOwn(desiredTeamMemberships, 'admin')) {
```

Comment:
Not sure that `desiredTeamMemberships` is relevant here; `admin` isn't a team. What's the case this is handling?